app/routers/email_send.py

```diff
@@ -0,0 +1,28 @@
+from app.database.database import get_db
```

Comment:
I think this file should be the general file to email API.

Change the name of this file to `email`